x/mint/keeper/keeper.go

```diff
@@ -176,6 +185,11 @@ func (k Keeper) DistributeMintedCoin(ctx sdk.Context, mintedCoin sdk.Coin) error
 	if err != nil {
 		return err
 	}
+
+	// Take the current balance of the developer rewards pool and remove it from the supply offset
```

Comment:
```suggestion
	// Take the current balance of the developer rewards pool and remove it from the supply offset
	// We add back the new supply at the end, in order to avoid any rounding discrepancies.
```